fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/query/AllGranularitySpec.groovy

```diff
@@ -26,6 +28,19 @@ class AllGranularitySpec extends Specification {
         writer.writeValueAsString(allGranularity) == /"all"/
     }
 
+    @Unroll
+    def "All granularity #relationship  #grain"() {
+
+        expect:
+        allGranularity.satisfiedBy(grain)
+        grain.satisfies(allGranularity)
+
+        where:
+
+        grain << ((DefaultTimeGrain.values() as List) + allGranularity)
+        relationship = " is satisfied by "
```

Comment:
Is this supposed to be the same every time? If so, should we just move it up into the description directly rather than being a param?